packages/react-dom/src/client/ReactDOMComponent.js

```diff
@@ -266,6 +266,7 @@ if (__DEV__) {
 export function ensureListeningTo(
   rootContainerInstance: Element | Node,
   reactPropEvent: string,
+  targetElement?: Element,
```

Comment:
Optional arguments can be pretty confusing to follow. Can we make it nullable but always pass it?